spec/preservation_ingest/update_catalog_spec.rb

```diff
@@ -18,10 +20,15 @@
     }
   end
   let(:mock_storage_object) do
-    instance_double(Moab::StorageObject, object_pathname: instance_double(Pathname), size: size, storage_root: strg_root, current_version_id: version)
+    instance_double(Moab::StorageObject, deposit_bag_pathname: deposit_bag_pathname, object_pathname: mock_moab_pathname, size: size, storage_root: strg_root, current_version_id: version)
   end
   let(:mock_storage_objects) { [mock_storage_object] }
 
+  let(:spec_dir) { File.join(File.dirname(__FILE__), '..') }
+  let(:deposit_dir_pathname) { Pathname(File.join(spec_dir, 'fixtures', 'deposit', 'complete-ingest')) }
```

Comment:
i'd consider changing `'complete-ingest'` to `'update-catalog'` or `'deposit_cleanup'`, but it also only lives for the duration of the test, so not a big deal either way

Reply:
Renamed.